doc/ref_impl/cntt-ri/chapters/chapter07.md

```diff
@@ -49,27 +59,51 @@ Logical steps for lab access and connectivity.
 
 Sample steps provided mimic those utilized for POD10 test lab access.
 
+This RI leverages OPNFV Pharos pod 15, which is hosted by Intel and requires VPN access.  Requests for VPN access must go through the OPNFV Infra Project.  Once on the VPN, two networks are reachable:
+
+1. Public Network - this is where the jump host is connected, and
+2. Admin Network - this is where OpenStack public enpoints and floating IP addresses can be reached
+
+Do we link to the yEd Network Topology diagram from here, or is this the right place to put it?
+
 <a name="7.5"></a>
 ## 7.5 Descriptor File Preparations
 
 Reference steps describing the use, creation, and implementation of descriptor files.
 
+This is where the Airship Mafiest files need to be documented:
```

Comment:
This document is going to be heavy on Airship information.  As an RI, do we want to keep it NVFI agnostic and reference an Airship cookbook with the specifics as an appendix?  That way as other NNVIs can be added later and this chapter will not get cluttered with "if this NVFI, do the following" type statements.

Reply:
@markbeierl - you're correct that we want to keep this "installer agnostic", so we should be able to call out what AirShip required (for the manifest/treasure-map) as generic requirements.  Then, to go a step further, since @opensource-tnbt (Sridhar) already has a cookbook for Airship, and a slide deck of steps/keylearnings/trouble-shooting, we include include that information as one of the installer options utilized, as a job-aid for anyone using this chapter for RI installs, with a link to the more detailed Airship cookbook steps.  Bottom line, both Generic and Specific would be added to the documentation.

Reply:
may i suggest something like (or similar to):
7.5 Available Installers  
    7.5.1 Airship
         7.5.1.1 Descriptor File Preparations]
         7.5.1.2 Deployment Installer & Install Steps]
    7.5.2 Future Installers
       * [7.5.2.1 Descriptor File Preparations]
       * [7.5.2.2 Deployment Installer & Install Steps]
* [7.6 Deployment Validations]

Reply:
Agree.  Reasonable.  The RI team (@fuqiao123 ) has put together lists of installer requirements.  For this documentation, @markbeier - this translates to Manifests and Treasure Map, which define the 'what'.  @opensource-tnbt , can you confirm you have this info for the recommended 7.5.1 section?  Then, this becomes a template for 7.5.2 for any/all installers.

Reply:
@opensource-tnbt , can you confirm you have this info for the recommended 7.5.1 section? Then, this becomes a template for 7.5.2 for any/all installers.